### DISCUSSION TITLE
Kobo eReader rejects PDFs claiming Adobe DRM protection

### DISCUSSION THREAD

**Discussion by ChristinMilloy** [elementdavv/internet_archive_downloader#47] [General]
Will it be possible to get PDF downloads via this extension that are DRM-Free?

**Comment by elementdavv** [comment]
No. It only downloads borrowed books in Internet Archive and full viewed books in HathiTrust.

**Comment by ChristinMilloy** [comment]
@elementdavv That's what confuses me -- The PDF books from Internet Archive open fine on my computer, but they're not compatible with my eReader.
